src/formatter-settings/types.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export interface JavaFormatterSetting {
+    id: string;
+    name?: string;
```

Comment:
Does `name` map to the label in the UI? When will it become undefined?

Reply:
Yes, will remove `?` to keep it always a valid value.